src/main/java/com/mabl/integration/jenkins/MablStepBuilder.java

```diff
@@ -134,6 +136,21 @@ private FilePath getOutputFileLocation(AbstractBuild<?, ?> build) {
         return fp;
     }
 
+    private EnvVars getEnvironmentVars(AbstractBuild<?, ?> build, BuildListener listener) {
+        final PrintStream outputStream = listener.getLogger();
+        EnvVars environmentVars = new EnvVars();
+        try {
+            environmentVars = build.getEnvironment(listener);
+        } catch (IOException e) {
+            outputStream.println(e.getMessage());
```

Comment:
We could add some logging detail like `Error gathering environmental variables`.